frontend/src/scenes/persons/Cohorts.tsx

```diff
@@ -85,6 +85,9 @@ function _Cohorts(): JSX.Element {
                 </span>
             ),
             render: function RenderCalculation(_, cohort: CohortType) {
+                if (cohort.is_static) {
+                    return '-'
```

Comment:
why '-'?

Reply:
Because last calculation isn't relevant for static cohorts. Happy to take other suggestions though?

Reply:
Ah, I was wondering whether this was a ts thing.

Suggestion: 'N/A' is used elsewhere in the app (paths, feature flags), doesn't matter much though.